.gitignore

```diff
@@ -8,3 +8,5 @@ venv/
 **__pycache__/
 **db.sqlite3
 media
+pillow
+black
```

Comment:
The entry `black` might be incorrect if you are trying to ignore files related to the Black code formatter. Ensure that you specify the correct path or pattern if you intend to ignore specific files or directories.